### PR TITLE
feat: use `webManifestUrl` on `injectDevManifest`

### PR DESCRIPTION
**Pull request by userquin**
I forgot to use `webManifestUrl` also on `injectDevManifest`: if someone using it will not work (`SvelteKit`  will not fail since the `manifest.webmanifest` is included on the `_layout`)

### CODE REVIEW

**Comment by userquin** [comment]
I need to add the content type when serving it, I'll fix it.

File: src/dev.ts [hunk 1]
```diff
@@ -58,13 +58,16 @@ export async function loadDev(id: string, options: ResolvedVitePWAOptions, viteC
...
+        // we need to exclude the manifest.webmanifest from the sw precache: avoid writing it to the dev-dist folder
+        const webManifestUrl = options.devOptions.webManifestUrl ?? `${options.base}${options.manifestFilename}`
         const { filePaths } = await generateServiceWorker(
           Object.assign(
             {},
             options,
             {
               workbox: {
                 ...options.workbox,
+                navigateFallbackDenylist: [new RegExp(`^${webManifestUrl}$`)],
```
Comment (antfu) [inline src/dev.ts:70]: Do we need to escape regex here?

Comment (userquin) [reply src/dev.ts:70]: no, using the constructor will replace with the escaped literal: `/manifest.webmanifest` => `^\/manifest.webmanifest$`.

I check it.

Comment (userquin) [reply src/dev.ts:70]: wait, I can double test...

Comment (userquin) [reply src/dev.ts:70]: It is ok:

imagen

**Pull request merged**